assets/scss/modules/_side-navigation.scss

```diff
@@ -75,6 +76,13 @@ Styleguide Side Navigation
   }
 }
 
+.side-nav__link--mobile-tall {
```

Comment:
The example in the comment above could cause specificify problems later due to nesting.
It's only taller at the mobile breakpoint, not beyond that. .side-nav--tall would suggest it's taller at all screen widths.